src/coreclr/src/pal/tests/palsuite/runpaltests.sh

```diff
@@ -19,11 +19,16 @@ echo "***** Testing PAL *****"
 echo
 
 # Store the location of the root of build directory
-BUILD_ROOD_DIR=$1
+BUILD_ROOT_DIR=$1
+
```

Comment:
nit:
```suggestion
BUILD_ROOD_DIR=$1
if [ -d "$(pwd)/$BUILD_ROOT_DIR" ]; then
  BUILD_ROOT_DIR="$(pwd)/$BUILD_ROOT_DIR"
fi
```

currently it does not work with the relative paths:

```sh
# does not work
$ src/coreclr/src/pal/tests/palsuite/runpaltests.sh artifacts//bin/coreclr/OSX.x64.Debug/paltests
# works
$ src/coreclr/src/pal/tests/palsuite/runpaltests.sh $(pwd)/artifacts//bin/coreclr/OSX.x64.Debug/paltests
```
both cases will work with the suggested change.

Reply:
Also, now it is a directory instead of path to txt file, could you please update this doc section: https://github.com/dotnet/runtime/blob/9e09fe3c263e6680ef4fa7aa0cbd36a477db3a7c/docs/workflow/testing/coreclr/unix-test-instructions.md#L91-L93
to:
<pre>
```sh
src/coreclr/src/pal/tests/palsuite/runpaltests.sh artifacts//bin/coreclr/$(uname).x64.Debug/paltests
# on macOS, replace $(uname) with OSX
```
</pre>